Makefile

```
@@ -41,79 +45,36 @@ fix: ## Fixup files in the repo.
-image-push-ansible-multiarch:
-	./hack/image/push-manifest-list.sh $(ANSIBLE_IMAGE) ${ANSIBLE_ARCHES}
-
-.PHONY: image-build-helm image-push-helm image-push-helm-multiarch
-HELM_CI_IMAGE = quay.io/operator-framework/helm-operator
-HELM_IMAGE ?= $(HELM_CI_IMAGE)
-HELM_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-image-build-helm: build
-	./hack/image/build-helm-image.sh $(HELM_CI_IMAGE):dev
-image-push-helm:
-	./hack/image/push-image-tags.sh $(HELM_CI_IMAGE):dev $(HELM_IMAGE)-$(shell go env GOARCH)
-image-push-helm-multiarch:
-	./hack/image/push-manifest-list.sh $(HELM_IMAGE) ${HELM_ARCHES}
+# Build operator-sdk project binaries
+build/scorecard-test build/scorecard-test-kuttl build/custom-scorecard-tests:
+	go build $(GO_GCFLAGS) $(GO_ASMFLAGS) -o $@ ./images/$(@F)
+build/%:
+	go build $(GO_BUILD_ARGS) -o $@ ./cmd/$*
 
-.PHONY: image-build-scorecard-test image-push-scorecard-test image-push-scorecard-test-multiarch
-SCORECARD_TEST_CI_IMAGE = quay.io/operator-framework/scorecard-test
-SCORECARD_TEST_IMAGE ?= $(SCORECARD_TEST_CI_IMAGE)
-SCORECARD_TEST_ARCHES:="amd64" "ppc64le" "arm64" "s390x"
-image-build-scorecard-test:
-	./hack/image/build-scorecard-test-image.sh $(SCORECARD_TEST_CI_IMAGE):dev
-image-push-scorecard-test:
-	./hack/image/push-image-tags.sh $(SCORECARD_TEST_CI_IMAGE):dev $(SCORECARD_TEST_IMAGE)-$(shell go env GOARCH)
-image-push-scorecard-test-multiarch:
-	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_IMAGE) ${SCORECARD_TEST_ARCHES}
+##@ Dev images
 
-.PHONY: image-build-scorecard-test-kuttl image-push-scorecard-test-kuttl image-push-scorecard-test-kuttl-multiarch
-SCORECARD_TEST_KUTTL_CI_IMAGE = quay.io/operator-framework/scorecard-test-kuttl
-SCORECARD_TEST_KUTTL_IMAGE ?= $(SCORECARD_TEST_KUTTL_CI_IMAGE)
-SCORECARD_TEST_KUTTL_ARCHES:="amd64" "ppc64le" "arm64"
-image-build-scorecard-test-kuttl:
-	./hack/image/build-scorecard-test-kuttl-image.sh $(SCORECARD_TEST_KUTTL_CI_IMAGE):dev
-image-push-scorecard-test-kuttl:
-	./hack/image/push-image-tags.sh $(SCORECARD_TEST_KUTTL_CI_IMAGE):dev $(SCORECARD_TEST_KUTTL_IMAGE)-$(shell go env GOARCH)
-image-push-scorecard-test-kuttl-multiarch:
-	./hack/image/push-manifest-list.sh $(SCORECARD_TEST_KUTTL_IMAGE) ${SCORECARD_TEST_KUTTL_ARCHES}
+# Convenience wrappers for building/pushing all images.
+.PHONY: image-build
+IMAGE_TARGET_LIST := operator-sdk helm-operator ansible-operator scorecard-test scorecard-test-kuttl custom-scorecard-tests
+image-build: $(foreach i,$(IMAGE_TARGET_LIST),image/$(i)) ## Build all images.
 
-.PHONY: image-build-custom-scorecard-tests
-CUSTOM_SCORECARD_TESTS_CI_IMAGE = quay.io/operator-framework/custom-scorecard-tests
-CUSTOM_SCORECARD_TESTS_IMAGE ?= $(CUSTOM_SCORECARD_TESTS_CI_IMAGE)
-image-build-custom-scorecard-tests:
-	./hack/image/build-custom-scorecard-tests-image.sh $(CUSTOM_SCORECARD_TESTS_CI_IMAGE):dev
+# Build an image.
+IMAGE_REPO ?= quay.io/operator-framework
+image/%: build/%
+	rm -rf build/_image && mkdir -p build/_image && mv build/$* build/_image
```

Any reason not to `cp` the binary? If I ran `make build image-build` I'd expect the binaries to exist in the build directory.

```suggestion
	rm -rf build/_image && mkdir -p build/_image && cp build/$* build/_image
```